internal/lookoutv2/repository/getjobs.go

```diff
@@ -70,14 +72,80 @@ type annotationRow struct {
 	annotationValue string
 }
 
-func NewSqlGetJobsRepository(db *pgxpool.Pool) *SqlGetJobsRepository {
+func NewSqlGetJobsRepository(db *pgxpool.Pool, useJsonbBackend bool) *SqlGetJobsRepository {
 	return &SqlGetJobsRepository{
-		db:            db,
-		lookoutTables: NewTables(),
+		db:              db,
+		lookoutTables:   NewTables(),
+		useJsonbBackend: useJsonbBackend,
 	}
 }
 
 func (r *SqlGetJobsRepository) GetJobs(ctx *armadacontext.Context, filters []*model.Filter, activeJobSets bool, order *model.Order, skip int, take int) (*GetJobsResult, error) {
+	if r.useJsonbBackend {
```

Comment:
Personally I'd create two private functions here, `getJobsLegacy` and `getJobsJsonB`,  and just have this function delegate based on `r.useJsonbBackend`.  Same for everywhere else we do this.

Reply:
I agree that it's a little silly here; this is the only place where the code for the two branches significantly, though (e.g., in `GroupJobs` we just make two different calls to `QueryBuilder` methods, and everything else is shared).